source/thisTarget.ts

```diff
@@ -102,5 +125,8 @@ export function initPrivateApi(): void {
     // also serves the purpose of throwing a specific error when no methods have been registered.
     // https://github.com/pixiebrix/webext-messenger/pull/80
     registerMethods({ __getTabData });
+
+    // Already includes per-context exclusion logic
+    void getTabInformation();
```

Comment:
👍 Comment could be clarified as "getTabInformation include per-context exclusion logic" to make it clear the comment refers to getTabInformation's behavior/pre-conditions